### PR TITLE
prefix all wonder-blocks-* packages with @khanacademy

### PR DESCRIPTION
**Pull request by kevinbarabash**
Need to do this before we can publish anything.

### CODE REVIEW

File: packages/wonder-blocks-button/package.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
   "private": true,
-  "name": "wonder-blocks-button",
+  "name": "@khanacademy/wonder-blocks-button",
   "version": "0.0.1",
   "design": "v1",
+  "publishConfig": {
+    "access": "public"
+  },
```
Comment (kevinbarabash) [inline packages/wonder-blocks-button/package.json]: This is needed so that the packages we publish are public.

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 46a0310d605d47034dcbb8399a62ae60dea6d15b

https://deploy-preview-120--wonder-blocks.netlify.com

File: demo/package.json [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
-    "wonder-blocks-color": "../packages/wonder-blocks-color",
-    "wonder-blocks-core": "../packages/wonder-blocks-core",
-    "wonder-blocks-grid": "../packages/wonder-blocks-grid",
-    "wonder-blocks-icon-button": "../packages/wonder-blocks-icon-button",
-    "wonder-blocks-link": "../packages/wonder-blocks-link",
-    "wonder-blocks-modal": "../packages/wonder-blocks-modal",
-    "wonder-blocks-progress-spinner": "../packages/wonder-blocks-progress-spinner",
-    "wonder-blocks-spacing": "../packages/wonder-blocks-spacing",
-    "wonder-blocks-typography": "../packages/wonder-blocks-typography"
+    "@khanacademy/wonder-blocks-button": "0.0.1",
```
Comment (kevinbarabash) [inline demo/package.json]: For this to work we'd need to add the `demo` folder to the list of `workspaces` in the root package.json but that breaks the test runner probably b/c `react@16` and/or `react-scripts` are hoisted up a level.  I need to update the demo to not use `react-scripts` and to use the same version of `react` but that'll have to be in a separate PR.

File: packages/wonder-blocks-button/package.json (hunk 1 above)
Comment (kevinbarabash) [reply packages/wonder-blocks-button/package.json]: It avoids having to re-authenticate which is impossible when running `lerna`.  You still need to do `npm login` before running `yarn run publish`.

File: packages/wonder-blocks-spacing/package.json [hunk 3]
```diff
@@ -1,19 +1,22 @@
 {
   "private": true,
-  "name": "wonder-blocks-spacing",
+  "name": "@khanacademy/wonder-blocks-spacing",
   "version": "0.0.1",
   "design": "v1",
+  "publishConfig": {
```
Comment (jeresig) [inline packages/wonder-blocks-spacing/package.json]: 🤷‍♂️ What does it mean when a package has a public publishConfig but is marked as `"private": true`?

File: demo/package.json (hunk 2 above)
Comment (jeresig) [reply demo/package.json]: Oof, that's annoying!

File: packages/wonder-blocks-button/components/button-core.js [hunk 4]
```diff
@@ -3,10 +3,14 @@ import React from "react";
 import {StyleSheet} from "aphrodite";
 import {Link} from "react-router-dom";
 
-import {LabelLarge, LabelSmall} from "wonder-blocks-typography";
-import Color, {SemanticColor, mix, fade} from "wonder-blocks-color";
-import {addStyle} from "wonder-blocks-core";
-import type {ClickableHandlers} from "wonder-blocks-core";
+import {LabelLarge, LabelSmall} from "@khanacademy/wonder-blocks-typography";
```
Comment (jeresig) [inline packages/wonder-blocks-button/components/button-core.js]: What commands did you run to do these replacements? Might be useful for the future!

Comment (kevinbarabash) [reply packages/wonder-blocks-button/components/button-core.js]: I did a simple find and replace.  I replaced `"wonder-blocks-` with `"@khanacademy/wonder-blocks-`.

File: packages/wonder-blocks-spacing/package.json (hunk 3 above)
Comment (kevinbarabash) [reply packages/wonder-blocks-spacing/package.json]: Yeah... the terminology is a little wonky. `"private": true` prevents publishing completely, `"publishConfig"` sets specific settings at publish time only, in this case the "access" setting does the following:
> When publishing scoped packages, the access level defaults to restricted. If you want your scoped package to be publicly viewable (and installable) set --access=public. The only valid values for access are public and restricted. Unscoped packages always have an access level of public.

**Comment by kevinbarabash** [comment]
> Seeing all these @khanacademy makes me wish we had @khan, but so be it :)

Part of me wishes we used `@khan` too b/c then it would go with our repo name.  We could potentially change our npmjs.org username to `khan` as well since it looks like someone is squatting on that username.

**Comment by MiguelCastillo** [comment]
the idea to try to claim `@khan` has been bounced around...  someone is just squatting it.  I am happy to start the process :)

https://www.npmjs.com/org/khan
https://www.npmjs.com/policies/disputes

**Comment by kevinbarabash** [comment]
I'll change `@khanacademy` to `@khan` before merging.

**Comment by kevinbarabash** [comment]
I changed my mind... I'm going stick with the `@khanacademy` prefix for now since we're already publishing packages under that prefix.
